config/env/all.js

```diff
@@ -1,6 +1,7 @@
 var path = require('path'),
 rootPath = path.normalize(__dirname + '/../..');
 var keys = rootPath + '/keys.txt';
+require('dotenv').config();
```

Comment:
You have place this in the wrong file. If you read the documentation for "_dotenv_" it should be called at the very earliest point. Which would make your "_server.js_" file an appropriate place to do this. This was also mentioned during your interaction session yesterday.